libs/arcade-cli/arcade_cli/main.py

```diff
@@ -376,6 +398,7 @@ def evals(
         "--no-tls",
         help="Whether to disable TLS for the connection to the Arcade Engine.",
     ),
+    debug: bool = typer.Option(False, "--debug", help="Show debug information"),
```

Comment:
```suggestion
    debug: bool = typer.Option(False, "--debug", "-d", help="Show debug information"),
```

Reply:
I didn't want to have a short command because this is more for us then customers... I hope.